service/src/main/java/uk/gov/hmcts/reform/fpl/model/ChildParty.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import uk.gov.hmcts.reform.fpl.enums.PartyType;
+import uk.gov.hmcts.reform.fpl.model.common.EmailAddress;
+import uk.gov.hmcts.reform.fpl.model.common.Party;
+import uk.gov.hmcts.reform.fpl.model.common.Telephone;
+
+import java.util.Date;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public final class ChildParty extends Party {
+    private final String gender;
+    private final String genderIdentification;
+    private final String livingSituation;
+    private final String livingSituationDetails;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    private final Date addressChangeDate;
```

Comment:
Ideally that should be LocalDate + we could configure the feature on Json serialiser to automatically take care of those so you don't need to put the pattern every time when dealing with the dates. But I guess to expedite this one it's not crucial to change it now.